src/combinatorics/quiver/iso.lean

```diff
@@ -0,0 +1,186 @@
+/-
+Copyright (c) 2023 Adam Topaz, Rémi Bottinelli. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Adam Topaz, Rémi Bottinelli
+-/
+import combinatorics.quiver.basic
+import combinatorics.quiver.cast
+import logic.equiv.basic
+import tactic.nth_rewrite
+/-!
+# Isomorphisms of quivers
+
+Isomorphisms of quivers, defined as pairs of prefunctors that compose to the identities.
+
+## Main definitions
+
+* For quivers `U` and `V`, `iso U V` is the type of isomorphisms between `U` and `V`, with
+  associated `iso.refl`, `iso.symm`, and `iso.trans` definitions.
```

Comment:
This sentence has very little informative content

Reply:
Indeed; I felt like I should write something, but didn't have much to say. Are empty module docstrings acceptable?

Reply:
Mentioning `quiver.iso` is good, but saying that "`iso.refl`, `iso.symm`, and `iso.trans` are associated definitions" is quite useless.